coveo-stew/README.md

```diff
@@ -232,24 +232,28 @@ Runs `stew build` to a temporary folder and ensures that pip is able to reinstal
 
 ## Custom Runners
 
-You can add any tool to `stew ci`:
+You can add any tool to `stew ci`, and you can also include your own autofix routines:
 
 ```
 [tool.stew.ci.custom-runners]
 flake8 = true
 bandit = { args = ["--quiet --recursive ."] }
+isort = { args = ["--check", "."], autofix = ["."] }
 ```
 
 ### Options
 
 The following options are supported for custom runners:
 
 - name: You can specify the CLI executable if it differs from the name of the tool.
-- args: Can be a string or a list of strings.
+- args: The arguments to invoke the check.
+- autofix: The arguments to invoke the autofix. Provide the empty string "" in order to run without arguments.
```

Comment:
Both `autofix` and `args` are a list of arguments and yet their naming differs quite a bit. I understand why you keep `args` as is since changing that would be a breaking change.

Maybe you could also provide a naming scheme that's more consistent. Something like:

```toml
isort = { check = ["..."], autofix = ["..."] }
isort = { check_args = ["..."], autofix_args = ["..."] }
```

To avoid a breaking change, you could still keep `args` around.

Reply:
I did exactly that.